data-host/library/src/popup/components/SendTokens.tsx

```diff
@@ -11,11 +11,13 @@ import {
 } from "@popup/components/Common";
 import { getProtocolService } from "@services/Factory";
 import { useState } from "react";
+import { useNavigate } from "react-router-dom";
 import styled from "styled-components";
 
 const FCL_UNIT = BigInt(10 ** 12);
 
-export function SendTokens(props: { onFinish: () => void }) {
+export function SendTokens(props: { onFinish: string }) {
+    const navigate = useNavigate();
```

Comment:
Run formatter.

Reply:
Formatter should always be applied, otherwise the CI would fail.